content/concepts/basics/08-actions.md

```diff
@@ -4,10 +4,42 @@ metaTitle: "sensenet - Actions"
 metaDescription: "sensenet Actions"
 ---
 
-# Actions
+Most of the operations done on content in sensenet is governed via Actions. An Action is basically a command, instructing the system to use a specific component, to get or update the addressed content item's properties. All of these actions can be invoked through OData REST API. There are many built-in ones and developers can publish custom operations as actions as well.
+
+Under the hood OData operations are integrated into the sensenet via the *Action Framework*. sensenet actions behave as OData operations. The action in Action Framework is the extensibility point: every 3rd party action appears automatically in OData metadata if the current user has enough permissions.
+
+![action menu](../img/actions.gif)
 
 # Built-in actions
 
-# Scenarios
+As it was mentioned in the introduction there are tons of built-in actions (operations) in sensenet. There are versioning related, permission related, user management related actions, there are many different use cases for them but the way they should be used is the same. They can be invoked on a content, have a unique name and can have optional a required parameters. An action call is always something similar, like the following:
```

Comment:
```suggestion
As it was mentioned in the introduction there are tons of built-in actions (operations) in sensenet. There are actions related to versioning, permission and user management. Use cases can be very different but the way they should be used is the same. They can be invoked on a content, have a unique name and can have optional and required parameters. An action call is always something similar, like the following:
```
can have optional **and** required parameters. --> itt nem vagyok biztos benne, hogy ez volt a gondolat amire átírtam.